hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl_invokeOnPartitionsTest.java

```diff
@@ -159,4 +132,34 @@ public void writeData(ObjectDataOutput out) throws IOException {
         public void readData(ObjectDataInput in) throws IOException {
         }
     }
+
+    private static class OperationImpl extends Operation {
+        private int response;
+
+        @Override
+        public void run() {
+            response = getPartitionId() * 2;
+        }
+
+        @Override
+        public Object getResponse() {
+            return response;
+        }
+    }
+
+    private static class SlowOperationImpl extends Operation {
```

Comment:
I would rename it to SlowOperation. It isn't an implementation of SlowOperation.

Reply:
Fixed